lib/package.gi

```diff
@@ -2259,6 +2259,13 @@ InstallGlobalFunction( ValidatePackageInfo, function( info )
     TestOption( record, "BannerString", IsString, "a string" );
     TestOption( record, "TestFile", IsFilename,
                 "a string denoting a relative path to a readable file" );
+    if not IsBound( record.TestFile ) then
+          Print( "#W  ******************************************************\n",
+                 "#W  No 'TestFile' is given but is highly recommended. Package\n",
```

Comment:
Shouldn't the message be wrapped to be not wider than row of asterisks ?

Reply:
I've extended the row of asterisks instead.